src/test/java/tests/URLTest.java

```diff
@@ -28,8 +28,9 @@ public URLTest(List<File> testFiles) {
                 org.checkerframework.checker.https.HTTPSChecker.class,
                 "url",
                 "-Anomsgtext",
-                "-Astubs=stubs/",
-                "-nowarn");
+                "-Astubs=Stubs/",
+                "-nowarn",
+                "-AprintErrorStack");
```

Comment:
There's no reason for this option other than debugging, so we should remove it from this test.